apps/collabora.sh

```diff
@@ -80,6 +80,21 @@ install_docker
 docker_prune_this 'collabora/code'
 docker_prune_this 'onlyoffice/documentserver'
 
+# remove OnlyOffice-documentserver if activated
+install_if_not jq
+if occ_command_no_check app:list --output=json | jq -e '.enabled | .documentserver_community' > /dev/null
+then
+    msg_box "Removing OnlyOffice-documentserver: $1\nYou will be given the option to abort when you hit OK."
+    any_key "Press any key to continue. Press CTRL+C to abort"
+    occ_command app:remove documentserver_community
+fi
+
+# Disable RichDocuments (Collabora App) if activated
+if [ -d "$NC_APPS_PATH"/richdocuments ]
+then
+    occ_command app:remove richdocuments
+fi
+
 # Disable RichDocuments (Collabora App) if activated
```

Comment:
disable it two times?

Reply:
that isn't the current state of this PR
Don't know why you have checked that out?